eqcorrscan/core/lag_calc.py

```diff
@@ -481,6 +483,10 @@ def lag_calc(detections, detect_data, template_names, templates,
         To generate a plot for every detection or not, defaults to False
     :type parallel: bool
     :param parallel: Turn parallel processing on or off.
+    :type plotdir: str
+    :param plotdir:
+        Path to plotting folder, plots will be output here in a folder of date
```

Comment:
It looks like the doc-strong doesn't match what directory is being created. Can you change this to match the code?